sdk/monitor/azure-monitor-opentelemetry/README.md

```diff
@@ -59,7 +59,7 @@ You can use `configure_azure_monitor` to set up instrumentation for your app to
 | Parameter | Description | Environment Variable |
 |-------------------|----------------------------------------------------|----------------------|
 | `connection_string` | The [connection string][connection_string_doc] for your Application Insights resource. The connection string will be automatically populated from the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable if not explicitly passed in. | `APPLICATIONINSIGHTS_CONNECTION_STRING` |
-
+| `logger_name` | The name of the [Python logger][python_logger] and all of it's children loggers that you want to collect telemetry for. | `N/A` |
```

Comment:
This sounds like the name should be a list of the children loggers, too. Not sure what a better phrasing would be.

Reply:
I'll remove the part about children loggers.